src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,411 @@
+
+- __Get the content of an existing request__
+
+GET /v4/requests/{requestUuid}
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | Required The unique identifier of the Request
+
+*Response:*
+
+[Request](#request)
+
+- __Update the content of an existing request__
+
+PUT /v4/requests/{requestUuid}
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | Required The unique identifier of the Request
+- | body | [Request](#request) | Required The content of the Request to update
+*Response:*
+
+[Request](#request)
+
+- __Delete an existing request__
+
+DELETE /v4/requests/{requestUuid}
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | Required The unique identifier of the Request
+
+*Response:*
+Response code
+
+- __Create an expense report linked to an approved request__
+
+POST /v4/requests/{requestUuid}/reports
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | Required The unique identifier of the Request
+
+*Response:*
+[ResourceLink](#resourcelink)
+
+
+#### <a name="workflow-resource"></a>Workflow resource
+
+Manage workflow transitions for a Request document.
+- __Move an existing request in the approval workflow__
+
+POST /v4/requests/{requestUuid}/{action}
+
+The HATEOAS links for actions available given the current user and state are listed in the operations of the Request resource.
+
+    Traveler actions
+        submit: initiate the approval workflow
+        recall: get back the Request, usually to modify the content
+        cancel: cancel the Request and attached itineraries
+        close: archive the Request
+        reopen: get back an archived Request
+    Approver / Processor actions
+        approve: move the Request to the next approval step
+        sendback: reject the Request and send back to the Traveler
+
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | __Required__ The unique identifier of the Request
+action | path | string | Required The state transition to be executed (submit, approve, recall, sendback, cancel, close or reopen)
+
+*Response:*
+[Request](#request-resource)
+
+#### <a name="expense-resource"></a>Expense resource
+
+Manage expected expense entries attached to a Request document.
+- __Create a new expected expense__
+
+POST /v4/requests/{requestUuid}/expenses
 
-### Parameters
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+requestUuid | path | string | __Required__ The unique identifier of the Request to which the expense is attached
+body | object | [Expense resource](#expense-resource) | __Required__ The expense content to create
```

Comment:
Missing "Format" column, even if empty for this type of resource
for the second parameter, the different values are linked to the respective column headers:
Name | Parameter Type | Data Type | Format | Description
- | body | object | [Expense](#expense-resource)